tests/virtual_machines_test.go

```diff
@@ -33,6 +36,24 @@ var _ = Describe("Virtual Machines", func() {
 	})
 
 	Context("Check the client", func() {
+		BeforeEach(func() {
+			// Verify that there are no VMs in the cluster
+			currentVMList := &kubevirtv1.VirtualMachineList{}
+			err := testClient.VirtClient.List(context.TODO(), currentVMList, &client.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Reading errors from blank expects is hard, please add expectations string:

`Expect(err).ToNot(HaveOccurred(), "Should successfully list add VMs")`

The same for other new Expects.

Reply:
done